### PR TITLE
map settings: accept touch events

### PR DESCRIPTION
**Pull request by sshane**
Related: https://github.com/commaai/openpilot/pull/28476

### CODE REVIEW

File: selfdrive/ui/qt/maps/map_settings.cc [hunk 1]
```diff
@@ -244,11 +249,11 @@ DestinationWidget::DestinationWidget(QWidget *parent) : QPushButton(parent) {
   inner_frame->setSpacing(0);
   {
     title = new ElidedLabel(this);
-    title->setAttribute(Qt::WA_TransparentForMouseEvents);
+//    title->setAttribute(Qt::WA_TransparentForMouseEvents);
```
Comment (sshane) [inline selfdrive/ui/qt/maps/map_settings.cc]: @incognitojam do you know why we need these?

**Comment by sshane** [comment]
@incognitojam this if statement made the function a bit confusing to read, think it's simpler like this. It mimics what the map widget does. Any reason it was done this way, or is this good?

File: selfdrive/ui/qt/maps/map_settings.cc (hunk 1 above)
Comment (incognitojam) [reply selfdrive/ui/qt/maps/map_settings.cc]: if you tap on the text the click event doesn't get passed to the DestinationWidget without it

**Comment by incognitojam** [comment]
If you test it and it seems to work fine then LGTM - can't think of any reason I did it this way

File: selfdrive/ui/qt/maps/map_settings.cc (hunk 1 above)
Comment (sshane) [reply selfdrive/ui/qt/maps/map_settings.cc]: https://github.com/commaai/openpilot/pull/28869/files

**Pull request merged**